.buildkite/pipeline.yml

```diff
@@ -114,6 +115,8 @@ steps:
         config: tests/composefiles/docker-compose.v2.1.yml
         commmand: ["/hello"]
 
+  - wait: ~
+
```

Comment:
Why did this change become necessary? I ask because forcing the build into multiple distinct chunks usually is disadvantageous in terms of build time, and historically we've been able to achieve this with dependency images in most places (except perhaps at the very end of the build).

Reply:
Sorry, I didn't set the base correctly. There is actually no way to set the base to `v4.16.0.1`, I'm not sure why... I changed it to the closest thing, which is off by one commit. But it can't be merged like that...